Desafio de projeto Power BI

thiagoaugustocosta opened the pull request:
Relatório - Thiago Costa

thiagoaugustocosta committed:
Desafio de projeto

Power BI - Thiago Costa

AGUSDEL123 commented:
Boa tarde,

Segue desafio módulo
[sample_financial.pdf](https://github.com/julianazanelatto/power_bi_analyst/files/12860392/sample_financial.pdf)
 1

AGUSDEL123 commented:
Bom dia,

Segue desafio 2 "Criando um Relatório de Vendas Elegante com Power BI"
[Desafio 2.pptx](https://github.com/julianazanelatto/power_bi_analyst/files/12871222/Desafio.2.pptx)

thiagoaugustocosta committed:
Desafio de projeto - Mysql + Azure

Fiz todo o passo a passo na criação de tabelas no BD Mysql Azure, mas quando tentei abrir no Power BI, não veio de jeito nenhum. Por isso, fiz apenas os prints do processo.

AGUSDEL123 commented:
Não fiz o desafio pois,

Porem, ja sei conectar a base no sql server e my sql, esse ai pensa numa aula demorada para o final não proseguir
